### PR TITLE
add swr

### PR DESCRIPTION
**Pull request by marcel-epp**
add swr and mongoose

### CODE REVIEW

**Comment by marcel-epp** [comment]
Please use the official domain: https://capstone-tripapp.vercel.app/
in the preview version, the dropdown are not working. I think this is a problem with mongobd integration.

File: components/Form.js [hunk 1]
```diff
@@ -23,14 +23,19 @@ export default function Form({
 
     if (data.activity && data.region === "disabled") {
       results = places.filter((place) => {
-        return place.activity.includes(data.activity);
+        return place.activitys.some(
```
Comment (MerleMak) [inline components/Form.js:26]: activities would be spelled with "ies" in the plural :)

File: components/Form.js [hunk 2]
```diff
@@ -23,14 +23,19 @@ export default function Form({
...
+        );
       });
     } else if (data.region && data.activity === "disabled") {
       results = places.filter((place) => place.region === data.region);
     } else if (data.region && data.activity) {
       results = places.filter(
         (place) =>
-          place.region === data.region && place.activity.includes(data.activity)
+          place.region === data.region &&
+          place.activitys.some(
```
Comment (MerleMak) [inline components/Form.js:36]:
```suggestion
          place.activities.some(
```

File: components/Form.js [hunk 3]
```diff
@@ -68,7 +73,7 @@ export default function Form({
           <StyledLabel htmlFor="activity"></StyledLabel>
           <StyledSelect name="activity" id="activity">
             <option value="disabled">---</option>
-            {getUniqueValues("activity").map((activity, id) => (
+            {getUniqueValues("activitys").map((activity, id) => (
```
Comment (MerleMak) [inline components/Form.js:76]:
```suggestion
            {getUniqueValues("activities").map((activity, id) => (
```

File: pages/api/places/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import dbConnect from "../../../db/connect";
+import Place from "../../../db/models/Place";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  if (request.method === "GET") {
+    const places = await Place.find().populate("activitys");
```
Comment (MerleMak) [inline pages/api/places/index.js:8]:
```suggestion
    const places = await Place.find().populate("activities");
```

**Review by MerleMak — COMMENTED** [review]
The code looks really solid!
Your Vercel preview doesn't seem to be working tho :)
Remember to add the env variable to the Vercel Project, to be able to access the database via the deployment!

Good job :))

**Pull request merged**